cmake/Zephyr-sdkConfig.cmake

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# This file provides Zephyr sdk config version package functionality.
+#
+
+# Those are Zephyr variables used.
+set(SDK_VERSION ${Zephyr-sdk_VERSION})
+set(SDK_MAJOR_MINOR ${Zephyr-sdk_VERSION_MAJOR}.${Zephyr-sdk_VERSION_MINOR})
+set(SDK_VERSION ${Zephyr-sdk_VERSION_MAJOR}.${Zephyr-sdk_VERSION_MINOR}.${Zephyr-sdk_VERSION_PATCH})
+set(SDK_MAJOR_MINOR_MICRO ${SDK_VERSION})
+
+get_filename_component(ZEPHYR_SDK_INSTALL_DIR ${CMAKE_CURRENT_LIST_DIR}/.. ABSOLUTE)
+set(ZEPHYR_SDK_INSTALL_DIR ${ZEPHYR_SDK_INSTALL_DIR})
+set(ZEPHYR_TOOLCHAIN_VARIANT zephyr)
+set(TOOLCHAIN_VENDOR zephyr)
+set(TOOLCHAIN_ARCH x86_64)
```

Comment:
Do we still need this?

Reply:
This is used here:
https://github.com/zephyrproject-rtos/sdk-ng/blob/c90442fc6875ac34167c0a2948893ec28d93ed4e/cmake/zephyr/host-tools.cmake#L3

but as this file is now moved out of Zephyr repo, and into Zephyr SDK, there is no reason for going through a variable, as it is not used elsewhere in Zephyr.

So will fix.

Reply:
Is this fix on the sdk-ng side or zephyr side?

Reply:
on sdk-ng side.
Fixed now.